qiskit_metal/renderers/utils.py

```diff
@@ -0,0 +1,35 @@
+from typing import List, Tuple
+from ..designs.design_base import QDesign
+
+
+def get_min_bounding_box(design: QDesign, qcomp_ids: List[int], case: int,
```

Comment:
Do you still need this method or will you use class BoundsForPathAndPolyTables() in https://github.com/Qiskit/qiskit-metal/blob/main/qiskit_metal/toolbox_metal/bounds_for_path_and_poly_tables.py#L9

Reply:
Will change this to include the new method

Reply:
Change done.